pcsx2/gui/MainFrame.cpp

```diff
@@ -584,19 +587,66 @@ void MainEmuFrame::OnActivate( wxActivateEvent& evt )
 }
 // ----------------------------------------------------------------------------
 
+static void GetMenuItemLabel(MenuIdentifiers Id, wxString* label)
```

Comment:
It requires less time (as I said before) and less thinking to understand the switch version IMO. But let's just disagree on this.